category_theory/natural_transformation.lean

```diff
@@ -81,8 +80,7 @@ def vcomp (α : F ⟹ G) (β : G ⟹ H) : F ⟹ H :=
 notation α `⊟` β:80 := vcomp α β
 
 @[simp] lemma vcomp_app (α : F ⟹ G) (β : G ⟹ H) (X : C) : (α ⊟ β) X = (α X) ≫ (β X) := rfl
-lemma vcomp_assoc (α : F ⟹ G) (β : G ⟹ H) (γ : H ⟹ I) : (α ⊟ β) ⊟ γ = (α ⊟ (β ⊟ γ)) := 
-begin ext, intros, dsimp, rw [assoc] end
+@[ematch] lemma vcomp_assoc (α : F ⟹ G) (β : G ⟹ H) (γ : H ⟹ I) : (α ⊟ β) ⊟ γ = (α ⊟ (β ⊟ γ)) := by tidy
```

Comment:
Did you mean to reintroduce `ematch` here?

Reply:
No! Thanks for catching this.